### PR TITLE
DOC: Fix typo in example for df.merge()

### PR DESCRIPTION
**Pull request by proinsias**

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -4174,23 +4174,7 @@ def append(self, other, ignore_index=False, verify_integrity=False):
         pandas.concat : General function to concatenate DataFrame, Series
             or Panel objects
 
-        Examples
-        --------
-
```
Comment (jreback) [inline pandas/core/frame.py]: why did you remove this?

**Comment by proinsias** [comment]
Whoops! Complete mis-typing in my editor. That will teach me to double-check, even for a minor edit!

**Comment by jreback** [comment]
thanks!
